auto_editor/AutoEditor.py

```diff
@@ -36,6 +36,8 @@ def make_changes(self) -> List[CodeChange]:
     def save_new_version(self, project: StructuredProjectSource):
         for path, code_lines in project.paths_to_lines.items():
             full_path = self.cta_version_root / path
+            Path(full_path.parent).mkdir(parents=True, exist_ok=True)
+            print('created')
```

Comment:
[minor]: I think it's better to either delete this print message or to add a bit more information so it can be useful for others